frontend/src/pages/post/[slug].tsx

```diff
@@ -50,7 +50,7 @@ export default function PostPage(props: Props) {
                     <p className="mt-4">
                         <i>
                             Dit artikel is een ingezonden opiniestuk. De opvattingen zijn daarmee geen officiële standpunten van ROOD en sluiten niet
-                            noodzakelijkerwijs aan op de meningen van al onze leden.
+                            noodzakelijkerwijs aan op de meningen van al onze leden. Leden kunnen opiniestukken indienen door een conceptversie te sturen naar de redactie via redactie@roodjongeren.nl
```

Comment:
Kan je het op een newline plaatsen en het een anchor maken, zodat het clickable is? Voeg ook de klassen "text-primary underline hover:cursor-pointer" toe op de link om hem goed te stylen.

Daarbij kan je ook je contactpersoon vragen om iets op de website te zetten, omdat alle contactpersonen hier ook toegang toe hebben. Misschien wil je hier nog iets over zeggen

Reply:
Ik heb helaas geen idee hoe ik het een anchor moet maken en de classes moeten toevoegen. Dat tweede is denk ik niet nodig op de site maar meer iets voor te communiceren naar de groepen.

Reply:
> Ik heb helaas geen idee hoe ik het een anchor moet maken en de classes moeten toevoegen. Dat tweede is denk ik niet nodig op de site maar meer iets voor te communiceren naar de groepen.

```suggestion
                            noodzakelijkerwijs aan op de meningen van al onze leden. 
                            Leden kunnen opiniestukken indienen door een conceptversie te sturen naar de redactie via <a class="text-primary underline hover:cursor-pointer" href="mailto:redactie@roodjongeren.nl>redactie@roodjongeren.nl</a>
```

Reply:
Ah cool thanks, heb hem gecommit